ui/src/app/cohort-search/search-bar/search-bar.component.tsx

```diff
@@ -182,61 +205,102 @@ export class SearchBar extends React.Component<Props, State> {
   componentDidMount() {
     document.addEventListener('click', (e) => {
       if (!!this.dropdown && !this.dropdown.contains(e.target as Node)) {
-        this.setState({options: null});
+        this.setState({ options: null });
       }
     });
   }
 
   componentDidUpdate(prevProps: Readonly<Props>): void {
-    const {node: {domainId}, searchTerms} = this.props;
+    const {
+      node: { domainId },
+      searchTerms,
+    } = this.props;
     if (searchTerms !== prevProps.searchTerms) {
-      if (domainId === Domain.PHYSICALMEASUREMENT.toString() || domainId === Domain.VISIT.toString()) {
-        triggerEvent('Cohort Builder Search - Physical Measurements', 'Search', searchTerms);
+      if (
+        domainId === Domain.PHYSICALMEASUREMENT.toString() ||
```

Comment:
This is the only stylistic choice that really jumped out at me. It's a price I'm willing to pay for consistency though.